obal/data/modules/koji_build.py

```diff
@@ -0,0 +1,69 @@
+"""
+Release a package to koji
+"""
+
+import re
+
+from ansible.module_utils.basic import AnsibleModule
+from ansible.module_utils.koji_wrapper import koji, KojiCommandError # pylint:disable=import-error,no-name-in-module
+from ansible.module_utils.obal import package_exists  # pylint:disable=import-error,no-name-in-module
+
+
+def main():
+    """
+    Build a package in koji
+    """
+    module = AnsibleModule(
+        argument_spec=dict(
+            srpm=dict(type='path', required=True),
+            scratch=dict(type='bool', required=False, default=False),
+            tag=dict(type='str', required=True),
+            nevr=dict(type='str', required=True),
+            package=dict(type='str', required=True),
+            tag_check=dict(type='bool', require=False, default=True),
+            koji_executable=dict(type='str', required=False)
+        )
+    )
+
+    tag = module.params['tag']
+    scratch = module.params['scratch']
+    srpm = module.params['srpm']
+    nevr = module.params['nevr']
+    package = module.params['package']
+    koji_executable = module.params['koji_executable']
+
+    if module.params['tag_check']:
+        if not package_exists(module.params['koji_executable'], tag, package):
+            module.fail_json(msg="Package has not been added to tag %s" % tag)
+
+    if not scratch:
+        try:
+            command = ['latest-build', '--quiet', tag, package]
+            build = koji(command, koji_executable)
+            build = build.split(' ')[0]
```

Comment:
I generally don't like changing variables this way since the type information changes.
```suggestion
            koji_output = koji(command, koji_executable)
            build = koji_output.split(' ')[0]
```